internal/.kratos.yaml

```diff
@@ -98,15 +104,37 @@ selfservice:
           hooks:
             -
               hook: revoke_active_sessions
+
     registration:
       ui_url: http://test.kratos.ory.sh/register
       lifespan: 98m
       after:
         default_browser_return_url: https://self-service/registration/return_to
         password:
           hooks:
-            -
-              hook: session
+            - hook: session
+            - hook: web-hook
```

Comment:
I would probably rename this to json-rpc :)

Reply:
Please, let us not do this. JSON-RPC is defined in https://www.jsonrpc.org/specification. it defines, how request and response payloads should look like. E.g. Request: `{"jsonrpc": "2.0", "method": "subtract", "params": [42, 23], "id": 1}`¸ Response: `{"jsonrpc": "2.0", "result": 19, "id": 1}`. This is not what we do, but we can easily mimic that by using a proper jsonnnet template. For a request it would look like: `function(flow, session) {jsonrpc:2.0, method: foo, params: flow.id}`. But the response content would anyway be ignored by the current implementation.

Reply:
Right, web-hook is probably common enough that people understand that this is a JSON request